packages/dd-trace/src/span_pointers.js

```diff
@@ -0,0 +1,46 @@
+const crypto = require('crypto')
+const SpanContext = require('../../dd-trace/src/opentracing/span_context')
+const id = require('../../dd-trace/src/id')
+
+const SPAN_LINK_KIND = 'span-pointer'
+const S3_PTR_KIND = 'aws.s3.object'
+
+// Span context with ids of 0 is used for span pointers
+const ZERO_CONTEXT = new SpanContext({
```

Comment:
did you confirm that all the code that does span pointer serialization is okay with these zero values?

Reply:
Yes, there's no validation for the trace/span id
https://github.com/DataDog/dd-trace-js/blob/5be631470047127fbb171162e94b3cb9fadf48a6/packages/dd-trace/src/format.js#L71-L91

Reply:
thanks!